config/manifests/bases/watcher-operator.clusterserviceversion.yaml

```diff
@@ -52,8 +52,9 @@ spec:
       kind: Watcher
       name: watchers.watcher.openstack.org
       specDescriptors:
-      - description: APIOverride, provides the ability to override the generated manifest
-          of several child resources.
+      - description: |-
+          APIOverride, provides the ability to override the generated
+          manifest of several child resources.
```

Comment:
im not sure why this was generated but I'm guessing this is that cause of the failure.

i did not update this myself it was one automatically so ill revert this out and see if that fixes the issue.